regression-test/suites/nereids_p0/create_table/test_create_table.groovy

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// this suite is for creating table with timestamp datatype in defferent 
+// case. For example: 'year' and 'Year' datatype should also be valid in definition
+
+suite("nereids_create_time_range_table") {
+    sql 'set enable_nereids_planner=true'
+    sql 'set enable_fallback_to_original_planner=false'
+    
+    def testTable = "test_time_range_table"
+
+    sql "DROP TABLE IF EXISTS ${testTable}"
+
+        // multi-line sql
+    def result1 = sql """
+        create table ${testTable} (
+		`actorid` varchar(128),
+		`gameid` varchar(128),
+		`eventtime` datetimev2(3)
+	)
+	engine=olap
```

Comment:
should cover all engine type, key type, index type, rollup type, partition def, column type, agg key type, etc.